colorizer_data/converter.py

```diff
@@ -105,6 +107,12 @@ def _make_frames_parallel(
     """
     nframes = len(grouped_frames)
     total_objects = get_total_objects(grouped_frames)
+
+    if math.isnan(total_objects) or (total_objects < 1):
```

Comment:
I paused a second while reading this line, wondering if `total_objects ==1` should work here. Then I traced back to `get_total_objects` function and understood the reasoning. Would a clearer variable name or a clarify comment help make this more intuitive?